libs/providers/launchdarkly-client/src/lib/launchdarkly-client-provider.spec.ts

```diff
@@ -139,13 +139,13 @@ describe('LaunchDarklyClientProvider', () => {
       });
 
       const res = ofClient.getBooleanDetails(testFlagKey, false);
-      expect(res).toEqual({
+      expect(res).toEqual(expect.objectContaining({
         flagKey: testFlagKey,
         flagMetadata: {},
         value: false,
         reason: 'ERROR',
         errorCode: 'TYPE_MISMATCH',
-      });
+      }));
```

Comment:
I've loosened a few brittle assertions with this PR, so that if new files are added/defaulted they don't break.